### PR TITLE
Fix #196, #146, Minimums, Approaching Minimums, and V1 Callouts, some EICAS warning changes

### LINKED ISSUE #196: Something is bleeding into the clock
Screenshot_20201220_182701
this is the captains clock. this does not happen with the other clock.

### PR DESCRIPTION
**Pull request by crazytimtimtim**
I noticed bunch of "warn: xtlua time overflow!" warnings, and I'd like it if you could check to see if you have the same problem.

### CODE REVIEW

**Comment by crazytimtimtim** [comment]
Is this too much in 1 pull request?

**Comment by crazytimtimtim** [comment]
Is this going to be merged? I guess I should test one last time first. Way too much for 1 pull request imo, makes things confusing.

**Comment by crazytimtimtim** [comment]
Is this going to be merged?

**Comment by oMrSmith** [comment]
Is it planned to have a "V1, rotate" callout?

**Comment by crazytimtimtim** [comment]
> Is it planned to have a "V1, rotate" callout?

"Rotate" is definitely a possibility, we just need someone to record it.
"V1" is already implemented in this pull request, and should come soon - that is, if @mSparks43 merges this...

**Comment by oMrSmith** [comment]
Which samplerate and bitdepth would you need the recording in?

**Comment by crazytimtimtim** [comment]
> Which samplerate and bitdepth would you need the recording in?

I don't think it's a big deal unless FMOD has some limitations, that's a question for @Matt726-S. I only do the coding, Matt does sounds.

**Comment by mSparks43** [comment]
why is numClimb being calculated twice in fltctls?

**Comment by crazytimtimtim** [comment]
> why is numClimb being calculated twice in fltctls?

I have no idea. I thought about changing it but decided to leave it as is.

I'll add another commit to fix, although I am unable to test.

**Comment by crazytimtimtim** [comment]
Ok seems like it's because it's a local variable, and `function B747_speedbrake_warn()`, which needs numClimb, is outside of the `B747_fltCtrols_EICAS_msg()` function.

@mSparks43 Do you think I should make it a dataref? I could see other stuff needing numClimb outside of fltCtrls.

**Comment by mSparks43** [comment]
> Do you think I should make it a dataref?

yes,


> is outside of the `B747_fltCtrols_EICAS_msg()` function

I see now.

**Comment by crazytimtimtim** [comment]
@mSparks43 done! Still unable to text unfortunately :/

**Comment by mSparks43** [comment]
I should be able to test and fix anything in a few hours if this is otherwise ready

**Review by crazytimtimtim — COMMENTED** [review]
Code _looks_ good to me, ready for you to test.

File: 747-400.acf [hunk 1]
```diff
@@ -146591,7 +146592,8 @@ GROUP Clocks
           CLICK_DX 0
           CLICK_DY 0
       GROUP Elapsed Time
-        SHOW_LESS 0.000000 laminar/B747/fltInst/capt/clock_et_chr_display
```
Comment (crazytimtimtim) [inline 747-400.acf:146594]: Not too sure what I did here though.

Comment (crazytimtimtim) [reply 747-400.acf:146594]: I changed it to `SHOW_EQUAL`, but not sure why.

**Comment by crazytimtimtim** [comment]
Possible conflicts with Marauder28's pull request? We both edited the acf, not sure how Github will handle that.

**Comment by crazytimtimtim** [comment]
I think I will close this and open separate pull requests. One for the callouts, one for the clocks, one for the EICAS warnings (and I'll include the LO/RF lower EICAS warnings.

**Comment by crazytimtimtim** [comment]
I'll keep this as a draft until I get all three pull requests in.

**Comment by crazytimtimtim** [comment]
I'll go ahead and close this, everything's either merged or in another pull request.

**Pull request closed without merging**